package.json

```diff
@@ -36,8 +36,13 @@
     "pegjs": "0.7.0",
     "request": "2.79.0",
     "underscore": "1.8.3",
-    "winston": "2.3.0",
-    "nodeunit": "0.10.2",
+    "winston": "2.3.0"
+  },
+  "devDependencies": {
```

Comment:
I was a little concerned about this, since SCL3 runs ``npm install --production``. I changed Dockerfile to run ``npm install --production``, and KumaScript appears to continue to work, so 👍